src/Common/CustomTagSelector/TagDetails.tsx

```diff
@@ -30,6 +30,7 @@ export const TagDetails = ({
     removeTag,
     tabIndex = 1,
     suggestedTagsOptions,
+    isHelmChart,
```

Comment:
instead of making it specific, we can make this generic like `hidePropagateTag` or something like this